e2e/tests/graph.spec.ts

```
@@ -27,230 +25,347 @@
-        const updateGraph = await graph.getGraphDetails();
-        expect(updateGraph[0].x).not.toBe(initialGraph[0].x);
-        expect(updateGraph[0].y).not.toBe(initialGraph[0].y);
-        await apicalls.removeGraph(graphName);
+        await graph.openDataPanelForElementInCanvas("Alice");
+        await graph.deleteGraphAttribute();
+        const response  = await apiCall.runQuery(graphName, "match (n) return n");
+        expect(response.result.data[0].n.properties).not.toHaveProperty('name');
+        await apiCall.removeSchema(graphName);
     });
 
+    test(`@readwrite validate deleting connection between two nodes updates canvas panels`, async () => {//working
```

ESLint

[error] Expected exception block, space or tab after '//' in comment.

(spaced-comment)